esp-hal/src/dma/buffers.rs

```diff
@@ -913,3 +913,54 @@ impl DmaRxStreamBufView {
         }
     }
 }
+
+static mut EMPTY: &mut [DmaDescriptor] = &mut [DmaDescriptor::EMPTY];
```

Comment:
I'm guessing this is guaranteed to be in internal RAM somehow.

There's probably too many `mut`s here.
```suggestion
static EMPTY: &mut [DmaDescriptor] = &mut [DmaDescriptor::EMPTY];
```
or
```suggestion
static mut EMPTY: [DmaDescriptor] = [DmaDescriptor::EMPTY];
```

Reply:
I'll let you figure out which one compiles out of these.

Reply:
I guess the array one does, but obviously not as an unsized slice.

Reply:
Ah lol, forgot about that.